io.github.nroduit.Weasis.desktop

```
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.0
+Name=Weasis
+Comment=Medical DICOM viewer
+Exec=io.github.nroduit.Weasis %U
+Icon=io.github.nroduit.Weasis.png
```

icon NEVER has an extension in desktop files.
```suggestion
Icon=io.github.nroduit.Weasis
```